.github/workflows/docker_scan.yml

```diff
@@ -39,7 +42,7 @@ jobs:
           exit-code: '1'
       - name: Upload CVE scan report
         if: always()
-        uses: actions/upload-artifact@v4
```

Comment:
I think we can probably keep the official `actions/*` at "flexible" versions (like v3, v4, etc).

Reply:
Yea, let's keep this as `v4`

Reply:
Done.